fewsats/core.py

```diff
@@ -114,6 +121,63 @@ def add_webhook(self:Fewsats,
     return self._request("POST", f"v0/users/webhook/add", json={"webhook_url": webhook_url})
 
 # %% ../nbs/00_core.ipynb 39
+@dataclass
+class FewsatsWebhookEvent:
+    offer_id: str
```

Comment:
Excellent, not related with this PR but soon we could send a field called "metadata" with things like billing information :eyes: